src/content.js

```diff
@@ -1,192 +0,0 @@
-/**
- * All of the items in the NugetFunctionality namespace indicate that the functions come from the Nuget Gallery JavaScript.
```

Comment:
@vladimyr one of the concerns I have is here -- I specifically mirrored the Nuget site's scripts for parity in case their way of applying the tabs changed in the future. This loses some of that mirroring. I agree it's much cleaner / better script overall, but I was specifically going for parity with the original site.

Do you think this could cause an issue down the road if things should change? I suppose it would just be slightly harder to adapt.